graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/internal/InputArgumentTest.kt

```diff
@@ -770,7 +770,7 @@ internal class InputArgumentTest {
         val exceptionWhileDataFetching = executionResult.errors[0] as ExceptionWhileDataFetching
         assertThat(exceptionWhileDataFetching.exception).isInstanceOf(ConversionFailedException::class.java)
         assertThat(exceptionWhileDataFetching.exception.message)
-            .contains("Failed to convert from type [java.util.LinkedHashMap<?, ?>] to type [@com.netflix.graphql.dgs.InputArgument java.lang.String] for value '{name=tester}'")
```

Comment:
Spring uses now `ObjectUtils.nullSafeConciseToString()` which returns the object hascode instead.

https://github.com/spring-projects/spring-framework/commit/e746230de67c382c12c1c3e2c8df251b0ba8b3ee